### PR TITLE
Update content.md

### PR DESCRIPTION
**Pull request by martab1994**
Fix typos, link and code bug

## What This PR Changes
- (Please explain here why you created the pull request and specify what it changes)

## Contribution Guidelines
- [ ] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/04.pro/boards/portenta-x8/tutorials/multi-protocol-gateway/content.md [hunk 1]
```diff
@@ -35,15 +35,15 @@ In this tutorial we will go through the steps on how to setup both the Linux and
 - USB-C cable (either USB-C to USB-A or USB-C to USB-C)
 - Wi-Fi Access Point with Internet Access
 - 868-915 MHz antenna with SMA connector
-- ADB or SSH. [Check how to connect to your Portenta X8](docs.arduino.cc/tutorials/portenta-x8/out-of-the-box#controlling-portenta-x8-through-the-terminal)
+- ADB or SSH. [Check how to connect to your Portenta X8](https://docs.arduino.cc/tutorials/portenta-x8/out-of-the-box)
```
Comment (marqdevx) [inline content/hardware/04.pro/boards/portenta-x8/tutorials/multi-protocol-gateway/content.md:38]: Why the anchor has been removed? 🤔

Comment (martab1994) [reply content/hardware/04.pro/boards/portenta-x8/tutorials/multi-protocol-gateway/content.md:38]: It was not working for me, it was giving 404. If you can make it work even better :)

Comment (marqdevx) [inline content/hardware/04.pro/boards/portenta-x8/tutorials/multi-protocol-gateway/content.md:38]:
```suggestion
- ADB or SSH. [Check how to connect to your Portenta X8](https://docs.arduino.cc/tutorials/portenta-x8/out-of-the-box#controlling-portenta-x8-through-the-terminal)
```

Comment (marqdevx) [reply content/hardware/04.pro/boards/portenta-x8/tutorials/multi-protocol-gateway/content.md]: sure

**Review by marqdevx — APPROVED** [review]
LGTM

**Pull request merged**